src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/controller/v1_0/JJDisputeController.java

```diff
@@ -273,7 +273,7 @@ public ResponseEntity<TicketImageDataJustinDocument> getTicketImageData(
 	 * @param id of the {@link JJDispute} to be deleted
 	 * @param ticketNumber of the {@link JJDispute} to be deleted
 	 */
-	@Operation(summary = "Deletes a particular JJDispute record.")
+	@Operation(summary = "Deletes a particular JJDispute record.", operationId = "DeleteJJDispute")
 	@ApiResponses({
```

Comment:
without this, could not generate the C# API from the swagger json, it was trying to use the name 1Async for this controller method

Reply:
That's interesting. Never had to add that to uniquely identify the operation. Maybe the OpenAPI spec was not able to identify it as a unique delete endpoint since it has only the base path. Good find!